release_dashboard/lib/enums/cherrypick.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Instead of creating an enums folder, you may just want a models folder. It's like creating a numbers folder.